opentracing-spring-web-itest/common/src/main/java/io/opentracing/mock/TextMapPropagator.java

```diff
@@ -0,0 +1,49 @@
+package io.opentracing.mock;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.Map;
+
+import io.opentracing.propagation.Format;
+import io.opentracing.propagation.TextMap;
+
+/**
+ * @author Pavol Loffay
+ */
+public class TextMapPropagator implements MockTracer.Propagator {
```

Comment:
Is this here while waiting for release of opentracing-java? If so, 0.20.8 is now available